modules/nixosConfigurations.linux-01/configuration.nix

```diff
@@ -20,7 +20,7 @@ in {
   hostName = "185.255.131.141";
 
   nix.settings.trusted-users = ["root" "sshsession"];
-  nix.settings.max-jobs = 1;
+  nix.settings.max-jobs = 0;
```

Comment:
i tried this but then the host cannot be updated if no remote builders are available. maybe there's a flag to force local build?

Reply:
yes, according to [the docs](https://nixos.org/manual/nix/stable/command-ref/conf-file.html?highlight=local#conf-max-jobs) we can override it in the build using `-j` again. and i just see that you've done exactly that above so it's all good :+1: